cloudify_ansible/utils.py

```diff
@@ -514,3 +519,61 @@ def _get_secure_values(data, sensitive_keys, parent_hide=False):
         config = _get_secure_values(config, config.get("sensitive_keys", {}))
         for key, value in config.items():
             _ctx.instance.runtime_properties[key] = value
+
+
+def make_virtualenv(path):
+    """
+        Make a venv for installing ansible module inside.
+    """
+    runner.run([
+        sys.executable, '-m', 'virtualenv', path
+    ])
+
+
+def install_packages_to_venv(venv, packages_list):
+    # Force reinstall in playbook venv in order to make sure
+    # they being installed on specified environment .
+    command = [get_executable_path('python', venv=venv), '-m', 'pip',
+               'install', '--force-reinstall'] + packages_list
```

Comment:
don`t think so.

Reply:
we want force-reinstall because  on 505 the core team appended to the PYTHONPATH envvar folders and then pip says requirement satisfied and not installing the package in the new venv.